runner/collect_results.py

```diff
@@ -10,8 +10,8 @@
 import json
 
 
-def dir_path(string):
-    if os.path.isdir(string):
+def src_path_type(string):
```

Comment:
This shouldn't change anymore right? Since `dir_path` now is an optional argument and we don't explicitly use 'latest'